Document delta saving feature. (#27)
craftcms/docs: extend/field-types.md

    // ...
}
```

## Supporting Delta Saves

> If your field type needs to do anything following [`afterElementSave()`](api:craft\base\Field::afterElementSave()) or [`afterElementPropagate()`](api:craft\base\Field::afterElementPropagate()), you can improve performance by skipping processing when the field’s value is unchanged.

Reviewer: ```suggestion
If your field type does any processing from [afterElementSave()](api:craft\base\FieldInterface::afterElementSave()) or [afterElementPropagate()](api:craft\base\FieldInterface::afterElementPropagate()), you can improve performance by skipping processing when the field’s value is unchanged.
```

> You can tell whether field content has changed by calling `isFieldDirty()` on the element.

Reviewer: ```suggestion
You can tell whether field content has changed by calling [isFieldDirty()](api:craft\base\ElementInterface::isFieldDirty()) on the element.
```

> This method checks for content changes before continuing to handle that element:

Reviewer: This line can probably be removed, as it’s not really adding anything.
